middlewares/body-validation/schema/followUserV2Schema.js

```diff
@@ -1,8 +1,6 @@
 const Schema = {
   user_id_followed: 'string|empty:false',
-  follow_source: 'string|empty:false',
-  username_follower: 'string|empty:false',
-  username_followed: 'string|empty:false'
+  follow_source: 'string|empty:false'
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes made to the `followUserV2Schema.js` file seem to be removing the `username_follower` and `username_followed` fields from the validation schema. This could potentially cause issues if these fields are still being used elsewhere in the codebase, or if they are expected in the request body for this endpoint. Please ensure that these fields are no longer required before removing them from the validation schema.
```diff
  const Schema = {
    user_id_followed: 'string|empty:false',
    follow_source: 'string|empty:false',
+   username_follower: 'string|empty:false',
+   username_followed: 'string|empty:false'
  };
  
  module.exports = Schema;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->